app/models/vacancy.rb

```diff
@@ -132,6 +132,19 @@ class Vacancy < ApplicationRecord
   extend VacancyFulTextSearchQueryModule
   scope :search_by_full_text, ->(query) { vacancy_full_text_search_query(query) }
 
+  scope :active_in_current_academic_year, lambda {
+    august_31st = Date.current.end_of_year.months_ago(4)
+    academic_year_end = if Date.current >= august_31st
+                          august_31st + 1.year
+                        else
+                          august_31st
+                        end
+    academic_year_start = academic_year_end - 1.year
```

Comment:
Isn't this off by one day? 
This will set the academic year start to `August 31st` and, according to the ticket, the start date should be `September 1st`.